doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -7,64 +7,179 @@ In this chapter, we introduce advanced commands to modify the way Coq
 parses and prints objects, i.e. the translations between the concrete
 and internal representations of terms and commands.
 
-The main commands to provide custom symbolic notations for terms are
-:cmd:`Notation` and :cmd:`Infix`; they will be described in the
-:ref:`next section <Notations>`. There is also a
-variant of :cmd:`Notation` which does not modify the parser; this provides a
-form of :ref:`abbreviation <Abbreviations>`. It is
-sometimes expected that the same symbolic notation has different meanings in
-different contexts; to achieve this form of overloading, Coq offers a notion
-of :ref:`notation scopes <Scopes>`.
-The main command to provide custom notations for tactics is :cmd:`Tactic Notation`.
+When inputing a term to Coq, it goes through several successive steps.
+
+* :gdef:`lexing`: the input is split in a linear stream of tokens, for
```

Comment:
```suggestion
* :gdef:`lexing`: the input is split into a series of tokens, for
```

If one is going to define a term "lexing" that appears in the glossary index, it should refer to the [Lexical Conventions](https://coq.gitlabpages.inria.fr/-/coq/-/jobs/4641769/artifacts/_build/default/doc/refman-html/language/core/basic.html#lexical-conventions) section rather than to this extremely terse definition.  The LC section has 3-4 pages of detailed information.

Reply:
I agree. However, `Lexical Conventions` only defines the tokens, not lexing itself. What should we do? I guess we should add a definition of lexing (it does not need to be much more than this sentence) in lexical conventions and refer to it here.